internals/example-app/serve.go

```diff
@@ -0,0 +1,102 @@
+package example_app
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui"
+	"github.com/secrethub/secrethub-cli/internals/secrethub/command"
+)
+
+type ServeCommand struct {
+	io ui.IO
+
+	host string
+	port int
+
+	appUsername string
+	appPassword string
+}
+
+// Register registers the command, arguments and flags on the provided Registerer.
+func (cmd *ServeCommand) Register(r command.Registerer) {
+	clause := r.Command("serve", "Runs the secrethub example by serving a web page.")
```

Comment:
And, with these future guides in mind, I have a slight preference for:
`secrethub run -- secrethub example-app start` 
Over:
`secrethub run -- secrethub example-app serve` 

Matches nicer with "wrap your app start command in `secrethub run`"

Reply:
The reason for choosing this command is that it:
1) Is clear about what it does: serve a page.
2) Allows you to later add different methods (like `print` or `console` which just dumps the output)